app/helpers/participants_helper.rb

```diff
@@ -6,4 +6,15 @@ def hall_of_fame_title
   def participant_badge_count(participant)
     participant.badge_acquisitions_count || 0
   end
+
+  def participant_image_profile_link(participant)
+    link_to edition_participant_path(@edition, participant).sub(/\d+\z/, participant.username) do
```

Comment:
@frmendes Updated it. Also removed `respond_with` from index and show methods in Participants Controller, can they be ignored in this case? I've tried to google it but I don't usually see it being used in this simples cases...